src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/HtmlEncoder.cs

```diff
@@ -24,7 +25,7 @@ public static HtmlEncoder Create(params UnicodeRange[] allowedRanges)
         }
     }
 
-    public sealed class DefaultHtmlEncoder : HtmlEncoder
+    internal sealed class DefaultHtmlEncoder : HtmlEncoder
```

Comment:
There's clearly a philosophical difference here :smile: Default on vs. default off - an age-old argument. I have no particular strong argument about this particular type. Just the decade-plus span of anecdotes of frequently regretting something being internal, and almost never regretting something being public.